public/app/features/alerting/unified/components/rules/SystemOrApplicationRules.tsx

```diff
@@ -17,6 +18,8 @@ export const SystemOrApplicationRules: FC<Props> = ({ namespaces }) => {
   const rules = useUnifiedAlertingSelector((state) => state.promRules);
   const rulesDataSources = useMemo(getRulesDataSources, []);
 
+  const filteredRules = useFilteredRules(namespaces);
```

Comment:
this should be at `RuleList.tsx` level; now filtering is not applied to threshold rules